### ISSUE TITLE
Is it tested with datasets?

### ISSUE THREAD

**Issue by xpcovertech** [aninyams/Auto-Insurance-Claim#1]
Hi,

I liked the approach you are taking for prediction. 
Have you run this algo for any dataset? If yes, how was the output?

**Comment by aninyams** [comment]
Hi there, 

Thank you for reaching out! Yes, I ran the algorithm on the Car Insurance Claim Dataset which is available on Kaggle. In terms of output and model performance, the output was pretty good. I used the Accuracy, AUC, Kappa, Sensitivity and Specificity to score the models and compared the results. The model with the highest accuracy was XGBoost, followed by neural network, then MARS. Please don’t hesitate to reach out if you have any other questions. 

Thank you!

**Comment by xpcovertech** [comment]
HI,

Thanks for your quick reply.
Can I use this for a project?
